src/com/scottyplunkett/server/DirectoryContent.java

```diff
@@ -0,0 +1,33 @@
+package com.scottyplunkett.server;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+class DirectoryContent {
+    private String build;
+
+    DirectoryContent(Path path){
+        String directory = String.valueOf(path);
+        build = build(directory);
+    }
+
+    String get(){
+        return build;
+    }
+
+    private String build(String directory) {
+        String content = "";
+        File[] files = new File(directory).listFiles();
+        List<String> names =  Arrays.asList(files).parallelStream()
+                                    .map(file -> file.getName())
+                                    .collect(Collectors.toList());
+        for (String name : names) {
```

Comment:
I think there's a reducer / fold operation you can use here to get these rolled up into a single string (using a `StringBuilder`?), might be worth looking into.